fastats/core/decorator.py

```diff
@@ -16,9 +16,8 @@ def code_transform(func, replaced):
     try:
         yield func
     finally:
-        for k, v in replaced.items():
-            func.__globals__[k] = v
-        replaced.clear()
+        # TODO : remove this context manager as it's now not needed?
+        pass
```

Comment:
Definitely not needed anymore

Reply:
Agreed - again, let's take the updated version from your PR